app.json

```diff
@@ -0,0 +1,8 @@
+{
+    "cron": [
+      {
+        "command": "/app/deploy/bin/backup.sh",
+        "schedule": "00 01 * * *"
+      }
+    ]
+  }
```

Comment:
This is how dokku says to do cron jobs...but I'm not convinced it will work with deployment via docker image rather than git push :shrug:

Reply:
Oh, interesting question. I guess we'll find out?

I suspect it might though, as *all* dokku apps are based running a docker container. It's just the the push way builds the container locally with a buildpack, whereas the `from-image` way uses a pre-built docker image.